### PR TITLE
[do not merge] Remove theme plugin

### LINKED ISSUE #159: Remove the theming-plugin
# Feature Request

## Description
Remove theming-plugin: https://github.com/cerner/terra-dev-site/blob/master/src/theming-plugin.js

This is no longer leveraged in our wepback and the preferred theme aggregation is the terra-toolkit [aggregate-themes pre-build script](https://github.com/cerner/terra-toolkit/tree/master/scripts/aggregate-themes).

Should be able to remove the `postcss` dependency as well:
https://github.com/cerner/terra-dev-site/blob/master/package.json#L89

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Resolves #159 

While cleaning up dependencies it was discovered terra button nor terra-slide-panel are used by terra-dev-site directly so I removed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,13 +86,11 @@
     "lodash.escaperegexp": "^4.1.2",
     "lodash.kebabcase": "^4.1.1",
     "lodash.startcase": "^4.4.0",
-    "postcss": "^6.0.21",
```
Comment (bjankord) [inline package.json]: This still may be needed for autoprefixer or the postcss-loader unless those pull in their own version of postcss.

Comment (emilyrohrbough) [reply package.json]: Terra-toolkit actually defines this

**Comment by mjhenkes** [comment]
All the Terra-dev-site major version bump work is being done here:
https://github.com/cerner/terra-dev-site/tree/application-layout-updates

Lets close this PR in favor of that branch.

**Comment by emilyrohrbough** [comment]
@mjhenkes 👍

**Pull request closed without merging**